google/cloud/accessapproval/doc/main.dox

```diff
@@ -78,6 +59,9 @@ The library automatically retries requests that fail with transient errors, and
 uses [exponential backoff] to backoff between retries. Application developers
 can override the default policies.
 
+## More Information
+
+- @ref common-error-handling - describes how the library reports errors.
```

Comment:
most of these ate the newline separating the content from the reference links :/